server/src/main/java/org/candlepin/dto/api/v1/JobStatusDTO.java

```diff
@@ -0,0 +1,468 @@
+        this.targetId = targetId;
+        return this;
+    }
+
+    /**
+     * Retrieves the owner id of this JobStatusDTO object.
+     *
+     * @return the owner id of this JobStatusDTO object.
+     */
+    public String getOwnerId() {
+        return ownerId;
+    }
+
+    /**
+     * Sets the owner id to set on this JobStatusDTO object.
+     *
+     * @param ownerId the owner id to set on this JobStatusDTO object.
+     *
+     * @return a reference to this DTO object.
+     */
+    public JobStatusDTO setOwnerId(String ownerId) {
+        this.ownerId = ownerId;
+        return this;
+    }
+
+    /**
+     * Retrieves the correlation id of this JobStatusDTO object.
+     *
+     * @return the correlation id of this JobStatusDTO object.
+     */
+    public String getCorrelationId() {
+        return correlationId;
+    }
+
+    /**
+     * Sets the correlation id to set on this JobStatusDTO object.
+     *
+     * @param correlationId the correlation id to set on this JobStatusDTO object.
+     *
+     * @return a reference to this DTO object.
+     */
+    public JobStatusDTO setCorrelationId(String correlationId) {
+        this.correlationId = correlationId;
+        return this;
+    }
+
+    /**
+     * Retrieves the result data of this JobStatusDTO object.
+     *
+     * @return the result data of this JobStatusDTO object.
+     */
+    public Object getResultData() {
+        return resultData;
+    }
+
+    /**
+     * Sets the result data to set on this JobStatusDTO object.
+     *
+     * @param resultData the result data to set on this JobStatusDTO object.
+     *
+     * @return a reference to this DTO object.
+     */
+    public JobStatusDTO setResultData(Object resultData) {
+        this.resultData = resultData;
+        return this;
+    }
+
+    /**
+     * Returns true if the job that this JobStatusDTO object represents is done. False otherwise.
+     *
+     * @return true if the job that this JobStatusDTO object represents is done. False otherwise.
+     */
+    public Boolean isDone() {
+        return done;
+    }
+
+    /**
+     * Sets true or false depending on if the job that this JobStatusDTO object represents is done.
+     *
+     * @param done if the job that this JobStatusDTO object represents is done.
+     *
+     * @return a reference to this DTO object.
+     */
+    public JobStatusDTO setDone(Boolean done) {
+        this.done = done;
+        return this;
+    }
+
+    /**
+     * Retrieves the URL path of this JobStatusDTO object.
+     *
+     * @return the URL path of this JobStatusDTO object.
+     */
+    public String getStatusPath() {
+        return this.statusPath;
+    }
+
+    /**
+     * Sets the URL path to set on this JobStatusDTO object.
+     *
+     * @param statusPath the URL path to set on this JobStatusDTO object.
+     *
+     * @return a reference to this DTO object.
+     */
+    public JobStatusDTO setStatusPath(String statusPath) {
+        this.statusPath = statusPath;
```

Comment:
This is something we can auto-generate like the getHref method in the Pool and Product DTOs.

Reply:
done